scripts/ensemble_obspace_diag.py

```diff
@@ -0,0 +1,320 @@
+import emcpy.utils.utils as utils
+import emcpy.stats.obspace
+from emcpy.plots.plots import LinePlot, HorizontalLine
+from emcpy.plots.create_plots import CreatePlot, CreateFigure
+import numpy as np
+import matplotlib
+
+matplotlib.use("Agg")  # Must be called before importing pyplot
+import matplotlib.pyplot as plt
+
+tic = utils.tic()
+
+expt_names = []
+# ********************************************************************
+#                        USER SPECIFIED PARAMETERS                  *
+# ********************************************************************
+n_mem = 30
+expt_names.append("rrfs_a_conus")
+# expt_names.append("just uncomment for a second experiment")
+
+date1 = "2022122000"
+date2 = "2022123023"  # will skip 00-18
+skip_enkf_hours = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18]  # right now rrfs only runs EnKF at 18-23Z
+
+datapath = "/lfs/h2/emc/da/noscrub/donald.e.lippi/rrfs_mon/diags/"  # rrfs_a_conus/"
```

Comment:
That would be nice, but I'm not certain it would make sense to do so. This script uses the RRFS ensemble diag files and there are 30 members, so for 1 day there would be about 150 files (considering just uv) since the ensemble is only run for a limited period each day. For reference, that would be 356M. I could save a subset of the ensemble though, but it is still going to be more than 1 file.

Let me know if you would still like me to add some data to point to locally.